src/test/java/com/microsoft/azure/spring/data/cosmosdb/repository/integration/ProjectRepositoryIT.java

```diff
@@ -158,4 +158,57 @@ public void testFindByWithOrPartition() {
         Assert.assertEquals(projects.size(), reference.size());
         Assert.assertEquals(projects, reference);
     }
+
+
+    @Test
+    public void testFindByWithAndOr() {
+        List<Project> projects = repository.findByNameAndCreatorOrForkCount(NAME_0, CREATOR_1, FORK_COUNT_2);
+        List<Project> reference = Arrays.asList(PROJECT_2);
+        Assert.assertEquals(projects, reference);
+
+        projects = repository.findByNameAndCreatorOrForkCount(NAME_1, CREATOR_1, FORK_COUNT_2);
+        Assert.assertEquals(projects.size(), 2);
```

Comment:
please enhance the test content, make sure the projects found is actually the same as reference, not only just compare their size.

Reply:
Also the test should cover both result with expected size and no result at all.